handsontable/handsontable
# Refactor HiddenColumns to be compatible with upcoming IndexMappers

### Context
As in title and in description of #6178. Here are necessary changes to adjust plugin to upcoming IndexMappers.

### How has this been tested?
- hide columns
- change columns order using eg.: ManualColumnMove
- insert/remove some columns
- everything should work consistently

### Types of changes
- [x] A non-breaking change

### Related issue(s):
1. #6178
